### PR TITLE
CT-EYB: Add paging to VETTEC approved programs table. #9722

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/9722

Based on https://github.com/department-of-veterans-affairs/vets-website/pull/12908

## Screenshots
Screen Shot 2020-06-04 at 9 39 38 AM
Screen Shot 2020-06-04 at 9 39 30 AM
Screen Shot 2020-06-04 at 9 39 25 AM
Screen Shot 2020-06-04 at 9 39 12 AM
Screen Shot 2020-06-04 at 9 41 47 AM
Screen Shot 2020-06-04 at 9 41 38 AM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by jhonny218 — APPROVED** [review]
LGTM

File: src/applications/gi/components/vet-tec/VetTecApprovedProgramsList.jsx [hunk 1]
```diff
@@ -1,103 +1,247 @@
...
+
+      return (
+        <div>
+          <button
+            type="button"
+            className="va-button-link learn-more-button"
+            onClick={this.handleShowMoreClicked}
+          >
+            Show next {showNextCount}
+            <i className="fas fa-chevron-down fa-xs vads-u-padding-left--1" />
```
[the remark this answers is not in the document]
Comment (bkjohnson) [reply src/applications/gi/components/vet-tec/VetTecApprovedProgramsList.jsx:174]: I think this needs `aria-hidden="true"`

[the remark this answers is not in the document]
Comment (zurbergram) [reply src/applications/gi/components/vet-tec/VetTecApprovedProgramsList.jsx]: added

**Review by elder-joseph — APPROVED** [review]

**Pull request merged**